hack/k8s/example-ro.template.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: datamon-ro-demo
+spec:
+  selector:
+    matchLabels:
+      app: datamon-ro-demo
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: datamon-ro-demo
+    spec:
+      containers:
+      - name: demo-shell
+        image: fuse-demo-shell:latest
+        imagePullPolicy: "IfNotPresent"
```

Comment:
I think this is best set to always? From the documentation ```imagePullPolicy is omitted and either the image tag is :latest or it is omitted: Always is applied.``` so the policy here will be ignored cause we want to pick the latest?